packit_service/worker/helpers/job_helper.py

```diff
@@ -165,14 +165,14 @@ def _report(
         so we can extend it in subclasses easily.
         """
         if self.is_gitlab_instance and not self.is_reporting_allowed:
+            login = self.project.service.user.get_username()
             description = (
-                f"{description}\n\n---\nPackit-User does not have access to the "
+                f"{description}\n\n---\nPackit does not have access to the "
                 "source project (=usually author's fork of the project). "
                 "(This is only about the representation of the results. "
                 "Packit is still able to do its job without having the permissions.)\n\n"
                 "*In case you wish to receive commit statuses instead of comments, please "
-                "add login of the author of this comment to your fork with a role "
-                "`Reporter`.*"
+                f"add {login} to your fork with a role `Developer`.*"
```

Comment:
would it make sense to link https://docs.gitlab.com/ee/user/permissions.html here and in the issue as well?

Reply:
I think it would be ideal to link our own docs… The fact that there are:

* Developer
* Reporter
* Maintainer

present basically means that we're not capable of keeping it in sync during “smallish” changes :D And it's definitely not an issue of the TODO list in PRs, cause I had problems “grepping” through the service code to find this hunk on which we're discussing right now :D I knew that we add this footer to the commit comments, but no keywords found it for me

Reply:
ok, then we should fix our docs and link them here, right?

Reply:
just going through the opened PRs and noticed this got forgotten, so what should we do here?

Reply:
I would avoid as much details as possible and just link the docs, which I think you have already updated?

Reply:
no, I did not touch the docs related to this PR